vk/backend/backend.go

```diff
@@ -143,20 +143,29 @@ func getContainerInfo(pod *v1.Pod) (string, error) {
 }
 
 func NewBackend(ctx context.Context, rp runtimeTypes.ContainerRuntimeProvider, pod *v1.Pod, cfg *config.Config, m metrics.Reporter) (*Backend, error) {
-	containerInfoStr, err := getContainerInfo(pod)
+	var containerInfo titus.ContainerInfo
```

Comment:
In the case of schema v1+, this containerInfo variable looks like it will remain empty, and passed to the backend.
I was trying to figure out, "if this variable is empty, then does the backend object even need it? And is it still empty by the time we write out the json file for the imds to use?

And I think I answer my on question with: Yes, it is ok to be empty, because when we create the container object, we don't even look at the containerinfo part if it is v1+.

And yes, I'm confusing the original pod.json that vk uses with the `containerConfigFile` json, which is written down later (which will our cinfo, no matter what, for imds to use).

Reply:
> And I think I answer my on question with: Yes, it is ok to be empty, because when we create the container object, we don't even look at the containerinfo part if it is v1+.

Yeah, that's correct.  I _think_ you're telling me that I should add a comment here, which I totally will. :)

Reply:
Hehe. Sure. All this should go away in <6 months (please), so don't sweat it.